tests/test_utils.py

```diff
@@ -113,3 +113,31 @@ def test_stops_checking_as_soon_as_the_first_file_fails(self):
                 utils.can_open_all_files_in_prefix(SOME_PREFIX, SOME_FILES)
             )
         self.assertEqual(can_open.call_count, 2)
+
+
+def test_path_translations():
+    paths = [
+        (r";z:\miniconda\Scripts\pip.exe",
+         ":/z/miniconda/Scripts/pip.exe",
+         ":/cygdrive/z/miniconda/Scripts/pip.exe"),
+        (r";z:\miniconda;z:\Documents (x86)\pip.exe;C:\test",
+         ":/z/miniconda:/z/Documents (x86)/pip.exe:/C/test",
+         ":/cygdrive/z/miniconda:/cygdrive/z/Documents (x86)/pip.exe:/cygdrive/C/test"),
+        # Failures:
+        # (r"z:\miniconda\Scripts\pip.exe",
+        #  "/z/miniconda/Scripts/pip.exe",
+        #  "/cygdrive/z/miniconda/Scripts/pip.exe"),
+
+        # ("z:\\miniconda\\",
+        #  "/z/miniconda/",
+        #  "/cygdrive/z/miniconda/"),
```

Comment:
I added these test cases to try and figure out what the giant regexes from `win_path_to_unix` and `unix_path_to_win` were doing. I'm a little surprised that these cases fail. Is this intended behavior?